### PR TITLE
chore: add get subsystem method to fetch individual subsystem

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**
Signed-off-by: Abhinandan Purkait <purkaitabhinandan@gmail.com>

### CODE REVIEW

File: nvmeadm/src/nvmf_subsystem.rs [hunk 1]
```diff
@@ -124,9 +152,29 @@ impl Subsystem {
         file.write_all(b"1").context(FileIoFailed { filename })?;
         Ok(())
     }
+
+    // Returns the particular subsystem based on the nqn and address.
```
Comment (tiagolobocastro) [inline nvmeadm/src/nvmf_subsystem.rs]: missing /

File: nvmeadm/src/error.rs [hunk 2]
```diff
@@ -23,6 +23,17 @@ pub enum NvmeError {
...
+    #[snafu(display(
+        "Nvmf subsystem with nqn: {}, host: {}, port: {} not found",
+        nqn,
+        host,
+        port
+    ))]
+    SubsystemNotFound {
+        nqn: String,
+        host: String,
+        port: String,
```
Comment (tiagolobocastro) [inline nvmeadm/src/error.rs]: u16?

**Review by tiagolobocastro — APPROVED** [review]

File: nvmeadm/src/nvmf_subsystem.rs (hunk 1 above)
Comment (Abhinandan-Purkait) [reply nvmeadm/src/nvmf_subsystem.rs]: Done.

File: nvmeadm/src/error.rs (hunk 2 above)
Comment (Abhinandan-Purkait) [reply nvmeadm/src/error.rs]: Oops. Thanks. Done

**Pull request merged**